grid/chat/get-messages/profile.supr

```diff
@@ -0,0 +1,153 @@
+name = "chat/get-messages"
+version = "1.0.0"
+
+"""
+GetMessages usecase
+Retrieve paginated list of messages
+"""
+usecase GetMessages {
+	input {
+		"""
+		Destination
+		Identification of destination to retrieve messages from
+		"""
+		destination! string!
+		
+		"""
+		AfterDate
+		Date that marks beginning of messages range until current time
+		"""
+		afterDate string
+        
+		"""
+		Limit
+		Maximum number of records to retrieve
+		"""
+		limit number
+		
+		"""
+		Page
+		Identification of page to retrieve
+		"""
+		page string
+	}
+	
+	result Messages
+	error Error
+
+	example Successfull {
+		input {
+			destination = 'general',
+			limit = 2
+		}
+
+		result {
+			messages = [
+				{
+					id = '1'
+					author = 'U01',
+					text = 'success!'
+				}
+			]
+		}
+	}
+
+	example Failed {
+		input {
+			destination = 'not-existing',
+		}
+
+		error {
+			title = 'channel_not_found'
+		}
+	}
+}
+
+model Message {
+	"""
+	Id
+	Identification of message
+	"""
+	id! string!
+
+	"""
+	Author
+	Author of retrieved message
+	"""
+	author! string!
+
+	"""
+	Destination
+	Destination of retrieved message
```

Comment:
Same as with input parameter. Also, do you need to return this if the user knows this upon input? :thinking: